app/services/filter_application_choices_for_providers.rb

```diff
@@ -27,7 +27,7 @@ def search(application_choices, candidates_name_or_reference)
       if candidate_ref_match
         application_choices.joins(:application_form).where('support_reference ILIKE ?', "#{candidate_ref_match[0]}%")
       else
-        application_choices.joins(:application_form).where("CONCAT(first_name, ' ', last_name) ILIKE ?", "%#{candidates_name_or_reference}%")
+        application_choices.joins(:application_form).where("CONCAT(first_name, ' ', last_name) ILIKE ?", "%#{candidates_name_or_reference.strip.squeeze(' ')}%")
```

Comment:
why are we `squeeze`ing? Won't this mangle user input?

Reply:
if it's only set to spaces, shouldn't it be ok? Could use `squish` instead which is more specific to spaces

Reply:
We've changed it now to `squish`, but the effect is the same. However, as this is in the search only, there's no actual modification to the user's input, only to how we compare their input to the data. I think we figured that there's never going to be a scenario where a user would get a more appropriate result if they put in additional spaces, as the way we format our full names in the search is by comparing against `first_name, ' ', last_name`